lib/foreman_tasks/tasks/test.rake

```diff
@@ -1,14 +1,16 @@
 namespace :test do
-  task :foreman_tasks => 'db:test:prepare' do
-    test_task = Rake::TestTask.new('foreman_tasks_test_task') do |t|
-      t.libs << ['test', "#{ForemanTasks::Engine.root}/test"]
-      t.test_files = Dir["#{ForemanTasks::Engine.root}/test/**/*_test.rb"]
-      t.verbose = true
-      t.warning = false
-    end
-
-    Rake::Task[test_task.name].invoke
+  desc 'Foreman Tasks plugin tests'
+  Rake::TestTask.new(foreman_tasks: 'db:test:prepare') do |t|
+    t.libs << Rails.root.join('test')
+    t.libs << ForemanTasks::Engine.root.join('test')
+    t.pattern = ForemanTasks::Engine.root.join('test', '**', '*_test.rb')
+    t.test_files = [Rails.root.join('test', 'unit', 'foreman', 'access_permissions_test.rb')]
```

Comment:
Should this be dropped so that we're consistent with how the test task is configured in other plugins?

Reply:
If you prefer, I can drop it and then we can solve it consistently with all plugins.

Reply:
Please do, unless we'll fix it everywhere this way